app/views/users/deletion_info.html.erb

```diff
@@ -26,28 +26,36 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<%= toolbar title: "#{l('account.deletion_info.heading', :name => h(@user.name))}" %>
-<%= labelled_tabular_form_for :user, :url => user_path(@user), :html => { :method => :delete, :class => 'confirm_required' } do |form| %>
-
+<%= labelled_tabular_form_for :user, :url => user_path(@user), :html => { :method => :delete, :class => 'confirm_required form danger-zone' } do |form| %>
```

Comment:
If you were to follow the [boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) in regards to the hash rockets that would be awesome. The transformation would look as follows:
Transform `:symbol => value` to `symbol: value`

You can also look [here](https://github.com/Azure7111/openproject/pull/3/files#diff-2c0f15451be4d9715f5b59749f4f84bfR52) for an example